pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/omstillingsstoenad/OmstillingsstoenadRevurderingFraser.kt

```diff
@@ -319,6 +344,45 @@ class OmstillingsstoenadRevurderingFraser {
         }
     }
 
+    object FeilutbetalingUtenVarselOpphoer : OutlinePhrase<LangBokmalNynorskEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+            title2 {
+                text(
+                    Language.Bokmal to "Feilutbetaling",
+                    Language.Nynorsk to "Feilutbetaling",
+                    Language.English to "Incorrectly payment",
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Fordi stønaden din er opphørt tilbake i tid, har du fått for mye " +
+                            "utbetalt. Vi oppretter en sak om mulig tilbakekreving av det du har fått for mye " +
+                            "utbetalt. Dette får du mer informasjon om i et eget brev fra oss.",
+                    Language.Nynorsk to "Ettersom stønaden din blei avvikla tilbake i tid, har du fått for " +
+                            "mykje utbetalt. Vi opprettar ei sak om mogleg tilbakekrevjing av det du har fått for " +
+                            "mykje utbetalt. Dette får du meir informasjon om i eit eige brev frå oss.",
+                    Language.English to "You have been overpaid because your Adjustment Allowance has been " +
+                            "terminated retroactively. We will start a case to assess whether you must repay " +
+                            "adjustment allowance. You will receive more information about this in a separate letter " +
+                            "from us. ",
+                )
+            }
+            paragraph {
+                text(
+                    Language.Bokmal to "Fordi stønaden din er opphørt tilbake i tid, har du fått for mye " +
+                            "utbetalt. Se vedlegg «Forhåndsvarsel - vi vurderer om du må betale " +
+                            "tilbake omstillingsstønad».",
+                    Language.Nynorsk to "Ettersom omstillingsstønaden din blei avvikla tilbake i tid, har du " +
+                            "fått for mykje utbetalt. Sjå vedlegget «Førehandsvarsel om eventuell tilbakekrevjing " +
+                            "av feilutbetalt omstillingsstønad».",
+                    Language.English to "You have been overpaid because your Adjustment Allowance has been " +
+                            "terminated retroactively. See the Attachment Advance notice – we are assessing whether " +
```

Comment:
Bør vi for Engelsk også referere til dokumentet med fnutter slik som for Nynorsk og Bokmål?

Reply:
Legger bare inn det som står i brevene og som kommer fra oversetterene. Regner med at det er en greie å ikke ha med fnutter på engelsk siden det ikke gjøres andre steder.